feature-group-definitions/individual-transforms.yml

```diff
@@ -0,0 +1,10 @@
+name: CSS translate, scale, and rotate properties
```

Comment:
These seem to be pretty well documented as "individual transform properties". How about using this as the name?

Reply:
I agree, that's a common naming of these that I've seen. My only concern is that if you know about the `scale` property, you'd have no way of finding it in a list of feature names where it's called "individual transform properties".

Ultimately I think this is an argument for both a structured/normalized name which would probably be "CSS translate, scale, and rotate properties" in this case, and a description which can spell it out.

@ddbeck what name do you think we should pick here?

Reply:
If this is well documented as "individual transform properties" then I think that wins. If we need the disambiguation or search-ability, perhaps we could append it in parentheses: "Individual transform properties (translate, scale, and rotate)"

But maybe we ought to get on with adding a description field, just for our own auto documenting practice.

```yaml
name: Individual transform properties
description: Transform elements with separate `translate`, `rotate`, and `scale` CSS properties.
```

Reply:
Looks good to me, I've updated with your suggestion verbatim.